oidc-ui/.env

```diff
@@ -45,4 +45,7 @@ REACT_APP_WALLET_LOGO_URL=""
 
 REACT_APP_CONSENT_SCREEN_TIME_OUT_BUFFER_IN_SEC=5
 
-REACT_APP_WALLET_QR_CODE_AUTO_REFRESH_LIMIT=3
\ No newline at end of file
+REACT_APP_WALLET_QR_CODE_AUTO_REFRESH_LIMIT=3
+
+# well known endpoints api list
+REACT_APP_WELL_KNOWN_ENDPOINT_API_LIST='[{"name":"OpenID Configuration","value":"/.well-known/openid-configuration"},{"name":"Jwks Json","value":"/.well-known/jwks.json"}]'
```

Comment:
if not used anywhere, then please remove this entry

Reply:
removed the variable